node-control.sh

```diff
@@ -8,6 +8,29 @@ function print_help {
   echo "Usage: $0 {attach|detach} <node-number>"
 }
 
+function wait_for_pg {
+    set +e
+    PG_HOST=$1
+
+    PSQL_UP=0
+    for i in {0..60}; do
+        pg_isready -h $PG_HOST
```

Comment:
another way to not have to disable the errors is to capture the return code:
xyz && retcode=$? || true

Reply:
Was a leftover from another script which uses `set -e`. Removed.